package.json

```diff
@@ -10,8 +10,21 @@
     "preview": "vite preview"
   },
   "dependencies": {
```

Comment:
You have a lot of component libraries here, and you don't seem to use most of them....